Jint.Tests/Runtime/EngineTests.cs

```diff
@@ -1883,6 +1883,34 @@ public void DateShouldParseToString()
             ");
         }
 
+
+        [Fact]
+        public void ShouldThrowErrorWhenMaxExecutionStackCountLimitExceeded()
+        {
+            new Engine(options => options.Constraints.MaxExecutionStackCount = 1000)
+                            .SetValue("assert", new Action<bool>(Assert.True))
+                            .SetValue("log", new Action<object>(Console.WriteLine))
+                            .Evaluate(@"

```

Comment:
maybe no console write lines, they just make test output bad - maybe just return needed data from `Evaluate`

Reply:
`log` removed. 
I used it for testing locally and did not expect to effect test output :)